### DISCUSSION TITLE
PHP-PM Integration

### DISCUSSION THREAD

**Discussion by john-everden** [clue/framework-x#121] [Ideas]
I think a nice solid docker image with hot reload, php-pm integration, easy access to php.ini settings and required modules setup would go a long with adoption. I'd be happy to contribute some effort if anyone else is interested.

**Comment by john-everden** [comment]
Is this threaded already and there isn't a need for PHP-PM or is this single thread beside what's forked?

**Reply by SimonFrings** [reply]
@john-everden Thanks for bringing this up :+1: 

Sounds nice!  Thanks to ReactPHP we can do a lot of things at once inside one thread. In the future we want to expand this behavior in Framework X for multiple threads which is going to be mighty powerful. If you think that there's an additional need for PHP-PM, you're always invited to create an integration for this!
